config/features.yml

```diff
@@ -811,10 +811,6 @@ features:
     actor_type: user
     description: enables function that removes toxic exposure data if user has opted out from toxic exposure condition on Form 526 submission
     enable_in_development: true
-  disability_526_toxic_exposure_opt_out_data_purge_by_user:
-    actor_type: user
-    description: enables function that removes toxic exposure data if user has opted out from toxic exposure condition on Form 526 submission
-    enable_in_development: true
```

Comment:
Once this is deployed, you'll need to follow step 4 of [this section of this guide](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Removingold/unusedfeaturetoggles) to completely remove this toggle. You'll need to do it in the terminal of each ArgoCD environment. If you or no one on your team has access to those terminals, let me know and I can do it for you. Remember, don't delete them in Argo until the code has been deployed.